source/js/main.js

```diff
@@ -11,7 +11,7 @@ import Petition from './components/petition/petition.jsx';
 import People from './components/people/people.jsx';
 import Takeover from './components/takeover/takeover.jsx';
 import MemberNotice from './components/member-notice/member-notice.jsx';
-import MultipageNav from './components/multipage-nav/multipage-nav.jsx';
+import MultipageNavMobile from './components/multipage-nav/multipage-nav.jsx';
```

Comment:
I wanted to rename this file to `multipage-nav-mobile` but didn't wanna do it in this PR just yet 'cuz that'd make code review hard.

Reply:
is there a follow-up filed already?

Reply:
not yet, will do after I wrap up this PR first

Reply:
follow-up ticket filed: https://github.com/mozilla/foundation.mozilla.org/issues/2364